app/react/App/DropdownMenu.tsx

```diff
@@ -0,0 +1,53 @@
+import { I18NLink, t } from 'app/I18N';
+import { Icon } from 'UI';
+import React, { useRef, useState, useCallback } from 'react';
+import { useOnClickOutsideElement } from 'app/utils/useOnClickOutsideElementHook';
+
+export type DropdownMenuProps = {
+  link: any;
```

Comment:
If posible, could we avoid the use of any? As it's basically just turning of TS.

Reply:
Removed any